index.bs

```diff
@@ -687,6 +689,16 @@ When the <dfn method for="XRFrame">getViewerPose(|referenceSpace|)</dfn> method
   1. If |frame|'s [=animationFrame=] boolean is <code>false</code>, throw an {{InvalidStateError}} and abort these steps.
   1. Let |pose| be a new {{XRViewerPose}} object.
   1. [=Populate the pose=] of |session|'s {{XRSession/viewerSpace}} in |referenceSpace| at the time represented by |frame| into |pose|.
+  1. If |pose| is <code>null</code> return <code>null</code>.
+  1. Let |xrviews| be an empty [=/list=].
+  1. For each [=view=] |view| in the [=XRSession/viewerSpace/list of views=] on the {{XRSession/viewerSpace}} of {{XRFrame/session}}, perform the following steps:
```

Comment:
To prevent this algorithm from getting too long and hard to follow, I think it would be good to move this for loop out into it's own algorithm which is referenced here. (ie: "Let |xrviews| be the [=list of XRViews=] for |frame|." or similar)

Reply:
This was initially the case, Nell asked me to inline it :smile:

Reply:
Slight misunderstanding.  I'm happy to have the view population part be a separate algorithm.  I wanted the looping to be done here, though.

Reply:
Oh wait, I mixed up two comments.  Yeah, I'm not super opinionated on this one.  It just didn't seem worth having a separate algorithm that wasn't ever invoked from more than one place.

Reply:
LOL. Sorry, don't want to ping-pong you between editors feedback. Just leave it as-is.